client/scss/components/forms/_field-comment-control.scss

```diff
@@ -40,22 +37,45 @@ $button-size: theme('spacing.5');
     .tab-content--comments-enabled .w-field__input:focus-within > &,
     &:hover,
     &:focus,
-    &.w-field__comment-button--focused {
+    &.w-field__comment-button--focused,
+    .w-field--admin_model_chooser.w-field--commentable:hover & {
```

Comment:
I don’t see what would match `w-field--admin_model_chooser`.

Reply:
oops this was a typo i've removed now